lib/features/result_page/presentation/widgets/more_info.dart

```diff
@@ -0,0 +1,88 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_markdown/flutter_markdown.dart';
+import 'package:qr_flutter/qr_flutter.dart';
+import 'package:share_plus/share_plus.dart';
+import 'package:threedpass/features/hashes_list/domain/entities/hashes_model.dart';
+import 'package:threedpass/features/hashes_list/presentation/widgets/hashes_primitive_list.dart';
+import 'package:threedpass/features/result_page/presentation/widgets/save_dialog.dart';
+
+class MoreInfo extends StatelessWidget {
+  final HashesModel hashesModel;
+  const MoreInfo({
+    required this.hashesModel,
+  });
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      crossAxisAlignment: CrossAxisAlignment.center,
+      children: <Widget>[
+        MarkdownBody(
+//                              styleSheet: MarkdownStyleSheet(
```

Comment:
I am voting towards removing all unused/dead code right away. We can restore this code via Git history if we like.

Reply:
Yes, the dead code should be removed. I used it to restore functionality

Reply:
Most of the unused code has already been removed. But in the [last](https://github.com/3Dpass/threedpass/pull/6/commits/370409783d4ed05a55d988076afb8fc911697d3b) commit I removed the rest.